### PR TITLE
Add a cron recipe dashboard page

### PR DESCRIPTION
**Pull request by rwcarlsen**
I think I got timezone stuff sorted finally.  Time issues in programming are a real headache!

### CODE REVIEW

File: civet/settings.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 # In other words, the hosts that this server will accept connections for
 # Note that this is required when DEBUG = False
 # Ex: ['localhost', 'www.moosebuild.org', 'moosebuild.org']
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['localhost','rod.inl.hpc.gov']
```
Comment (milljm) [inline civet/settings.py:46]: remove this change

File: civet/apps.py [hunk 2]
```diff
@@ -24,6 +24,7 @@ def schedulePinger():
 
         # Formats time to something more suitable for logs (no time zone or ms
         format_time = lambda t : t.strftime("%Y-%m-%d %H:%M:%S")
+        local_tz = pytz.timezone("US/Mountain")
```
Comment (milljm) [inline civet/apps.py:27]: 👍

**Review by milljm — DISMISSED** [review]
Just need to remove the change to ALLOWED_HOSTS.

File: civet/settings.py (hunk 1 above)
Comment (rwcarlsen) [reply civet/settings.py]: done

File: civet/apps.py (hunk 2 above)
Comment (milljm) [inline civet/apps.py:27]: 👍

File: ci/views.py [hunk 3]
```diff
@@ -34,6 +34,9 @@
 from django.views.decorators.cache import never_cache
 from ci.client import UpdateRemoteStatus
 import os, re
+from datetime import datetime
+from croniter import croniter
+import pytz
```
Comment (milljm) [inline ci/views.py:39]: Is this something we need to add to requirements.txt?

**Review by milljm — COMMENTED** [review]
Looks good and ready. Just need the go ahead about that python timezone possibly being listed in requirements.txt

File: ci/views.py (hunk 3 above)
Comment (rwcarlsen) [reply ci/views.py:39]: croniter and pytz are already in it.  datetime is just the standard library.

Comment (milljm) [reply ci/views.py:39]: My mistake, I didn't see pytz (was the only one I had in question)

**Pull request merged**